src/custom/state/orders/updater.ts

```diff
@@ -62,6 +62,8 @@ function _computeFulfilledSummary({
       // We only have the API order info, let's at least use that
       summary = `Swap ${sellToken} for ${buyToken}`
     }
+  } else {
+    console.log(`computeFulfilledSummary::API data not yet in sync with blockchain`)
```

Comment:
ubernit: If you really want to leave this console, I would give a name like `[state:orders:updater] computeFulfilledSummary`

Reply:
Sure, will update